lib/perl/Genome/Model/CwlPipeline/Command/PrepForTransfer.pm

```diff
@@ -0,0 +1,82 @@
+package Genome::Model::CwlPipeline::Command::PrepForTransfer;
+
+use strict;
+use warnings;
+
+use Genome;
+
+class Genome::Model::CwlPipeline::Command::PrepForTransfer {
+    is => 'Command::V2',
+    has => [
+        builds => {
+            is => 'Genome::Model::Build::CwlPipeline',
+            is_many => 1,
+            doc => 'The build(s) to prepare for data transfer',
```

Comment:
Two of the parameter doc lines end in `.` and this one does not.  (I'd probably take them out of the other two, but consistency is good either way :smile:)